feat(service-account): Adds support for TLSPC service account

rvelaVenafi opened the pull request:
Closes VC-31982

rvelaVenafi committed:
feat(service-account): Adds support for TLSPC service account

Closes VC-31982

rvelaVenafi committed:
feat(service-account): Adds support for TLSPC service account

Closes VC-31982

luispresuelVenafi commented on examples/tlspc_service_account/main.tf:
```diff
@@ -0,0 +1,47 @@
+output "cert_private_key" {
+  sensitive = true
+  value = venafi_certificate.dev_certificate.private_key_pem
+}
```
Nit: could you add a newline here?

rvelaVenafi committed:
fix(service-account): Changes deprecated pkcs12 encode function

Changes deprecated pkcs12.Encode() function in favor of pkcs12.LegacyRC2.Encode() which is the recommendation from developers.

Closes VC-31982
